docs/source/posts/2024/2024-06-06-week-1-robin.rst

```diff
@@ -0,0 +1,106 @@
+Week 1: It officially begins...
+===============================
+
+.. post:: June 06 2024
+   :author: Robin Roy
+   :tags: google
+   :category: gsoc
+
+Hi, I'm `Robin <https://github.com/robinroy03>`_ and this is my blog about week 1.
+
+My goal for week 1 was to start with `Retrieval-Augmented Generation (RAG) <https://www.pinecone.io/learn/retrieval-augmented-generation/>`_, check different databases and host every endpoint. My week1 and week2 are very intertwined because I applied everything I did during week1 on week2. (I'm writing this blog midway through week2)
+
+why phi-3-mini-4k-instruct?
```

Comment:
Cool, you explain very well why you chose the phi-3-mini-4k-instruct model. In my opinion your blog would be even more accessible if you could add a few short sentences to explain technical terms. 

For example, what is a "contextual model with 3.8 billion parameters" and why is it important ?

Reply:
I see you've also put links for technical terms to blogs explaining these terms, but these links aren't visible unless you hover over them with the mouse, which isn't possible on the mobile version of the site.

Perhaps you could put a section at the beginning with all the links to these terms that will be discussed in the blog.